plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2621,7 +2664,7 @@ public void registerCoreMembers() {
         //
         // @Description
         // Add or remove a notable object for that can be used to reference objects such as in events.
-        // TODO: Document Command Details
+        // Notable objects are "permanent" versions of other dObjects. (See: <@link language Particle Effects>)

```

Comment:
... particle effects?

Also I would fix the above sentence because it's confusing as hell
`for that can be used to reference objects such as in events` lol